### PR TITLE
fix normalize

### PR DESCRIPTION
**Pull request by jperez999**
This PR is related to #1455. The normalize operator was using an itemsize 64 output dtype clashing with the other defaults of 32. We change to itemsize 32 in order to allow integration tests to pass.

### CODE REVIEW

**Comment by karlhigley** [comment]
Looks like this change breaks the TF4Rec test with a dtype discrepancy over there instead 🤔

File: ci/build_and_test.sh [hunk 1]
```diff
@@ -11,7 +11,7 @@ python -m pip install --user --upgrade pip setuptools wheel pybind11 numpy==1.20
 python -m pip uninstall nvtabular merlin-core -y
 python -m pip install --user --upgrade --no-cache-dir merlin-core@git+https://github.com/NVIDIA-Merlin/core.git
 python -m pip install --user natsort==8.1.0
-python setup.py develop --user --no-deps
+python setup.py develop
```
Comment (benfred) [inline ci/build_and_test.sh:14]: This is breaking CI - do you need to make this change?

File: nvtabular/ops/normalize.py [hunk 2]
```diff
@@ -94,16 +94,19 @@ def supports(self):
         )
 
     def clear(self):
+        """Clear previously recorded stats"""
```
Comment (benfred) [inline nvtabular/ops/normalize.py:97]: Can we just use the docstrings from the parent class? I'd rather not maintain two copies of the same content just to make interrogate happy

**Review by benfred — APPROVED** [review]

**Comment by jperez999** [comment]
Not the correct fix

**Pull request closed without merging**